tasks/install_powershell.ps1

```diff
@@ -24,10 +24,23 @@ catch [System.Management.Automation.CommandNotFoundException] {
   }
 }
 
+function Test-PuppetInstalled {
+  Get-WmiObject -Class Win32_Product | sort-object Name | select Name | where { $_.Name -match "Puppet Agent"}
```

Comment:
Can we not just call the powershell and bash code for the version task instead of having separate logic for detecting a puppet agent?

Reply:
Updated to use `version_powershell.ps1` to detect if an agent is installed. The `install_bash.sh` task could also be updated to use `version_bash.sh` to detect if an agent is installed, but I think that'd be better suited for another ticket since it would require a change to both tasks.

Reply:
Yeah dealing with JSON in bash is a pain. Copying over the one-liner from the shell script is OK for now. I think really what we would want is to make this a plan. But again, another day another ticket...

It looks like the travis failure is unrelated to this work, I will let someone from nightswatch decide if they want to merge this before that is fixed.